modules/navier_stokes/test/tests/finite_volume/ins/mms/channel-flow/cylindrical/cartesian-version/2d-rc-no-slip-walls.i

```diff
@@ -0,0 +1,248 @@
+    variable = u
+    rho = ${rho}
+    momentum_component = 'x'
+  []
+  [u_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = u
+    mu = ${mu}
+    momentum_component = 'x'
+  []
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = u
+    momentum_component = 'x'
+    pressure = pressure
+  []
+  [u_forcing]
+    type = INSFVBodyForce
+    variable = u
+    functor = forcing_u
+    momentum_component = 'x'
+  []
+
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = v
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = v
+    mu = ${mu}
+    momentum_component = 'y'
+  []
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = v
+    momentum_component = 'y'
+    pressure = pressure
+  []
+  [v_forcing]
+    type = INSFVBodyForce
+    variable = v
+    functor = forcing_v
+    momentum_component = 'y'
+  []
+[]
+
+[FVBCs]
+  [u_walls]
+    type = INSFVNoSlipWallBC
+    variable = u
+    boundary = 'left right'
+    function = 'exact_u'
+  []
+  [v_walls]
+    type = INSFVNoSlipWallBC
+    variable = v
+    boundary = 'left right'
+    function = 'exact_v'
+  []
+  [p]
+    type = INSFVOutletPressureBC
+    variable = pressure
+    function = 'exact_p'
+    boundary = 'top'
+  []
+  [inlet_u]
+    type = INSFVInletVelocityBC
+    variable = u
+    function = 'exact_u'
+    boundary = 'bottom'
+  []
+  [inlet_v]
+    type = INSFVInletVelocityBC
+    variable = v
+    function = 'exact_v'
+    boundary = 'bottom'
+  []
+[]
+
+[Functions]
+  [exact_u]
+    type = ParsedFunction
+    value = 'sin(x*pi)*cos(y*pi)'
+  []
+  [forcing_u]
+    type = ADParsedFunction
+    value = '2*pi^2*mu*sin(x*pi)*cos(y*pi) - 2*pi*rho*sin(x*pi)*sin(y*pi)*cos(1.3*x)*cos(y*pi) + 2*pi*rho*sin(x*pi)*cos(x*pi)*cos(y*pi)^2 + 1.5*cos(1.5*x)*cos(1.6*y)'
+    vars = 'mu rho'
+    vals = '${mu} ${rho}'
+  []
+  [exact_v]
+    type = ParsedFunction
+    value = 'cos(1.3*x)*cos(y*pi)'
+  []
+  [forcing_v]
+    type = ADParsedFunction
+    value = '1.69*mu*cos(1.3*x)*cos(y*pi) + pi^2*mu*cos(1.3*x)*cos(y*pi) - 1.3*rho*sin(1.3*x)*sin(x*pi)*cos(y*pi)^2 - 2*pi*rho*sin(y*pi)*cos(1.3*x)^2*cos(y*pi) + pi*rho*cos(1.3*x)*cos(x*pi)*cos(y*pi)^2 - 1.6*sin(1.5*x)*sin(1.6*y)'
+    vars = 'mu rho'
+    vals = '${mu} ${rho}'
+  []
+  [exact_p]
+    type = ParsedFunction
+    value = 'sin(1.5*x)*cos(1.6*y)'
+  []
+  [forcing_p]
+    type = ParsedFunction
+    value = '-pi*rho*sin(y*pi)*cos(1.3*x) + pi*rho*cos(x*pi)*cos(y*pi)'
+    vars = 'rho'
+    vals = '${rho}'
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = 'NEWTON'
+  petsc_options_iname = '-pc_type -pc_factor_shift_type -pc_factor_mat_solver_type'
+  petsc_options_value = 'lu       NONZERO               superlu_dist'
+  line_search = 'none'
+  nl_rel_tol = 1e-12
+  nl_abs_tol = 1e-12
+[]
+
+[Outputs]
+  exodus = true
+  csv = true
+  [dof]
+    type = DOFMap
+    execute_on = 'initial'
+  []
+[]
```

Comment:
```suggestion
[Outputs]
  csv = true
[]
```

Reply:
I'm guessing this was on an individual commit, and it's registering as outdated so I can't commit the suggestion :-(